sdk/core/core-amqp/samples/cbsAuth.ts

```diff
@@ -52,11 +53,15 @@ export const connectionContext = ConnectionContextBase.create(parameters);
  */
 export async function authenticate(
   audience: string,
-  closeConnection = false
+  closeConnection: boolean
 ): Promise<CbsResponse> {
   await connectionContext.cbsSession.init();
-  const tokenObject = await connectionContext.tokenProvider.getToken(audience);
-  const result = await connectionContext.cbsSession.negotiateClaim(audience, tokenObject);
+  const tokenObject = await connectionContext.tokenCredential.getToken(audience);
+  const result = await connectionContext.cbsSession.negotiateClaim(
+    audience,
+    tokenObject!,
```

Comment:
If tokenObject was null, then throw an error. Then remove the ! here. Same for other samples